assembl/static2/js/app/components/common/ideaPreview.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Link } from 'react-router';
+import { Translate } from 'react-redux-i18n';
+import Statistic from './statistic';
+
+class IdeaPreview extends React.Component {
```

Comment:
Pure component means the render method is not called when the prev props and next props are the same. This is not the case for functional component, render is always called.